src/views/Stack/Card.tsx

```diff
@@ -638,6 +638,11 @@ export default class Card extends React.Component<Props> {
         : this.handleGestureEventHorizontal
       : undefined;
 
+    const overrideFlex =
+      cardStyle && cardStyle.hasOwnProperty('flex')
```

Comment:
This won't work since `style` prop can be an array, `null`, `undefined`, `false` etc.

Reply:
Oops. Will fix.

Reply:
I've changed the code to recognize and handle array. It automatically takes the last defined `flex` as it overwrites them when progressing, which is I believe the correct way handle that.

Values that falsify were already skipped and it stays that way.